Index: libs/cocos2d/CCDirector.m
===================================================================
--- libs/cocos2d/CCDirector.m	(revision 311)
+++ libs/cocos2d/CCDirector.m	(working copy)
@@ -220,10 +220,10 @@
 	/* calculate "global" dt */
 	[self calculateDeltaTime];
 
-	accumDtt += dt;
-
 	/* tick before glClear: issue #533 */
 	if( ! isPaused_ ) {
+		accumDtt += dt;
+
 		//PATCH: QUITARTS fixed time
 		while (accumDtt > DT) {
 			[[CCScheduler sharedScheduler] tick:DT];	
